storage/clickhouse.go

```
@@ -183,78 +180,50 @@ func (s *ClickHouseStorage) StoreProcessedLogs(ctx context.Context, logs ...enti
-		return querier.QueryResponse{}, fmt.Errorf("failed to build query: %w", err)
+		return nil, fmt.Errorf("cannot construct query: %w", err)
 	}
 
-	// Execute the query
-	rows, err := s.conn.Query(ctx, result.Query, result.Args...)
+	rows, err := s.conn.Query(ctx, q, args...)
 	if err != nil {
-		return querier.QueryResponse{}, fmt.Errorf("failed to execute query: %w", err)
+		return nil, fmt.Errorf("cannot query database: %w", err)
 	}
 	defer rows.Close()
```

golangci-lint (2.11.4)

[error] Error return value of `rows.Close` is not checked

(errcheck)